x/concentrated-liquidity/README.md

```diff
@@ -228,13 +228,50 @@ b) Having the front end round the tick's actual price to the nearest human reada
 
 One draw back of this implementation is the requirement to create many ticks that will likely never be used. For example, in order to create ticks at 10 cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$ value of -5 must be set, requiring us to traverse ticks 1-3600000 before reaching _$10,000_. This should simply be an inconvenience and should not present any valid DOS vector for the chain.
 
-### Scope of Concentrated Liquidity
+# Chosing an Exponent At Price One Value
```

Comment:
Could you please elaborate how does base quote translate to the choice of token0 vs token1? That is, if I want token 0 to be SHIB, wich one of the examples should I use?

Reply:
Added which asset was asset0 and asset1 here https://github.com/osmosis-labs/osmosis/pull/4919/commits/8556ae63f5c692f8daa0efbc57f358453c3d1202